core/src/serve_repair.rs

```diff
@@ -40,46 +41,83 @@ pub const MAX_ORPHAN_REPAIR_RESPONSES: usize = 10;
 pub(crate) const REPAIR_PEERS_CACHE_CAPACITY: usize = 128;
 // Limit cache entries ttl in order to avoid re-using outdated data.
 const REPAIR_PEERS_CACHE_TTL: Duration = Duration::from_secs(10);
+pub const MAX_ANCESTOR_RESPONSES: usize = 25;
```

Comment:
Should be able to bump to 30. `1232 - 4 (nonce) - 4 (response version enum discriminator) - 4 (slot_hash length) == 1220` and each slot hash is 40 bytes

Reply:
Can you also add a comment on how you came to the number 25 (or 30 if you change it)

Reply:
Made things a lot more explicit here: https://github.com/solana-labs/solana/pull/18681/commits/c9816c36e8cd6fa091b7c66feb98474c7d94ab30